README.md

```diff
@@ -1,3 +1,22 @@
 # Rulma
 
 React components for [Bulma](https://github.com/jgthms/bulma)
+
+## Getting started
+#### Install
+```bash
+npm install --save rulma
+```
```

Comment:
I don't know. Feels unnecessary? I think Yarn users should be able to cope quite well even with current installation instructions?